Sources/Transifex/Core.swift

```diff
@@ -226,36 +224,45 @@ class NativeCore : TranslationProvider {
                    params: [String: Any],
                    context: String?) -> String {
         var translationTemplate: String?
-        let localeToRender = localeCode ?? self.locales.currentLocale
-        let isSource = self.locales.isSource(localeToRender)
+        let localeToRender = localeCode ?? locales.currentLocale
+        let key = txGenerateKey(sourceString: sourceString,
+                                context: context)
+
+        translationTemplate = cache.get(key: key,
+                                        localeCode: localeToRender)
         
-        /// If the source locale is requested, or if the source string is missing from cache,
-        /// the bypass localizer is used, to look up on the application bundle and fetch the
-        /// localized content for the source locale (if found) by bypassing swizzling.
+        var applyMissingPolicy = false
         
-        if isSource {
-            translationTemplate = self.bypassLocalizer.get(sourceString: sourceString,
-                                                           params: params)
-        }
-        else {
-            let key = txGenerateKey(sourceString: sourceString,
-                                    context: context)
-            translationTemplate = cache.get(key: key,
-                                            localeCode: localeToRender)
-            if !String.containsTranslation(translationTemplate) {
-                let bypassedString = self.bypassLocalizer.get(sourceString: sourceString,
-                                                              params: params)
-                
-                return missingPolicy.get(sourceString: bypassedString)
+        /// If the source string is not found in the cache, use the bypass localizer to look it up on the
```

Comment:
`If the source string` -> should be `If the string`

Reply:
Done!